x-pack/metricbeat/module/azure/database_account/_meta/data.json

```diff
@@ -0,0 +1,40 @@
+{
+    "@timestamp" : "2020-02-25T08:53:00.000Z",
+    "cloud" : {
+        "provider" : "azure",
+        "region" : "westeurope"
+    },
+    "event" : {
+        "module" : "azure",
+        "duration" : 4877063600,
+        "dataset" : "azure.database_account"
+    },
+    "metricset" : {
+        "name" : "database_account",
+        "period" : 300000
+    },
+    "azure" : {
+        "dimensions" : {
+            "databasename" : "testdb"
+        },
+        "database_account" : {
+            "timegrain" : "PT5M",
```

Comment:
is this field mapped by `fields.yml`?, we may need to add an entry

Reply:
perhaps we should move it to `azure.timegrain`? as it's common to all metricsets

Reply:
indeed, I have moved `timegrain` inside the `azure` object as this is a common property for all metricsets